test/parallel/test-timers-active-handles/deasync.cc

```diff
@@ -0,0 +1,18 @@
+#include <node.h>
+#include <uv.h>
```

Comment:
@trevnorris The test here merely has the name of the module, there are no module deps.